### ISSUE TITLE
Add new/blank line at end of output?

### ISSUE THREAD

**Issue by gingerbeardman** [tldr-pages/tldr-cpp-client#28]
I noticed a new/blank line is added before output for clarity.
So I propose that one be added afterwards for the same reason.

current:
```
MacBookPro-Matt:~ matt$ tldr shasum

shasum

Calculate or check cryptographic SHA checksums.

- Calculate the SHA1 checksum for a file:
    shasum filename

- Calculate the SHA256 checksum for a file:
    shasum --algorithm 256 filename

- Calculate the SHA512 checksum for multiple files:
    shasum --algorithm 512 filename1 filename2

- Check a file with a list of sums against the directory's files:
    shasum --check list_file

- Calculate the SHA1 checksum from stdin:
    somecommand | shasum
MacBookPro-Matt:~ matt$
```

proposed:
```
MacBookPro-Matt:~ matt$ tldr shasum

shasum

Calculate or check cryptographic SHA checksums.

- Calculate the SHA1 checksum for a file:
    shasum filename

- Calculate the SHA256 checksum for a file:
    shasum --algorithm 256 filename

- Calculate the SHA512 checksum for multiple files:
    shasum --algorithm 512 filename1 filename2

- Check a file with a list of sums against the directory's files:
    shasum --check list_file

- Calculate the SHA1 checksum from stdin:
    somecommand | shasum

MacBookPro-Matt:~ matt$
```

config:
> tldr v1.3.0 (v1.3.0)
> Copyright (C) 2016 Arvid Gerstmann
> Source available at https://github.com/tldr-pages/tldr-cpp-client

**Comment by yonezzzet** [comment]
I agree with your proposal.
So I created PR #29 !

**Comment by gingerbeardman** [comment]
@yonezzzet thank you!
